game.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include "randomEngine.cpp"
+#include "player.cpp"
+
+class NumberGame{
+    int randNumber;
+    RandomEngine randomEngine;
+    Player human;
+    Player pc;
+public:
+    NumberGame() : human(), pc(), randomEngine(1, 20){}
+    void run(){
+        while(human.is_playable() && pc.is_playable()) {
```

Comment:
If pc does not have money for contunie game, i want to get message;
The PC does not have enogth money to play. You win this game.
If human does not have money for contunie game, i want to get message;
You do not have enought money to play. PC win this game.

Also, each new round i want to see 'Round started' instead of "game begin".

When you start application it could be showned 'Game Being' and in the end when nobody has money show: Game finished